client/extensions/woocommerce/app/store-stats/index.js

```diff
@@ -58,9 +59,41 @@ class StoreStats extends Component {
 			date: unitSelectedDate,
 			limit: 10,
 		};
-		const widgets = [ topProducts, topCategories, topCoupons ];
+		// console.log(`queryDate ${ queryDate }`);
+		// console.log(`unitQueryDate ${ unitQueryDate }`);
+		// console.log(`selectedDate ${ selectedDate }`);
+		// console.log(`unitSelectedDate ${ unitSelectedDate }`);
+		// console.log(`endSelectedDate ${ endSelectedDate }`);
+		const topWidgets = [ topProducts, topCategories, topCoupons ];
 		const widgetPath = `/${ unit }/${ slug }${ querystring ? '?' : '' }${ querystring || '' }`;
 
+		const widgetList1 = (
+			<div className="store-stats__widgets-column spark-widgets" key="sparkwidgets1">
+				<WidgetList
+					siteId={ siteId }
+					header={ null }
+					emptyMessage={ translate( 'No data found.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'No data found' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/sparkline-widget-tables)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).